packages/renderer/src/stores/kubernetes-deployments.ts

```diff
@@ -0,0 +1,21 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { buildKubernetesResourceStore } from './kubernetes-resource';
+
+export const kubernetesDeployments = buildKubernetesResourceStore('deployments');
```

Comment:
Finally, this is not what I would like to do, as this store will be created at startup. I'll change to call the build function from svelte templates, and dispose the associated resources when the component is destroyed

Reply:
I'm removing the stores part for now, as this change will need some updates on the `EventStore` class (make it Disposable essentially)